postgres/src/tokio/test_utils.rs

```diff
@@ -0,0 +1,254 @@
+use crate::schema::{TableId, TableName};
+use crate::tokio::options::PgDatabaseOptions;
+use tokio::runtime::Handle;
+use tokio_postgres::{Client, NoTls};
+
+pub struct PgDatabase {
+    pub options: PgDatabaseOptions,
+    pub client: Client,
+}
+
+impl PgDatabase {
+    pub async fn new(options: PgDatabaseOptions) -> Self {
+        let client = create_pg_database(&options).await;
+
+        Self { options, client }
+    }
+
+    /// Creates a new publication for the specified tables.
+    pub async fn create_publication(
+        &self,
+        publication_name: &str,
+        table_names: &[TableName],
+    ) -> Result<String, tokio_postgres::Error> {
+        let table_names = table_names
+            .iter()
+            .map(TableName::as_quoted_identifier)
+            .collect::<Vec<_>>();
+
+        let create_publication_query = format!(
+            "CREATE PUBLICATION {} FOR TABLE {}",
+            publication_name,
+            table_names.join(", ")
+        );
+        self.client.execute(&create_publication_query, &[]).await?;
+
+        Ok(publication_name.to_string())
```

Comment:
We can avoid returning the publication name string here as the caller already knows it.